### PR TITLE
also hide inhibited alerts

### PR DESCRIPTION
**Pull request by caarlos0**
refs #1034

I feel like it would be better to rename the form parameter to something else, since it's not only silenced alerts anymore... what do you folks think?

### CODE REVIEW

**Comment by brancz** [comment]
Internally this is often referred to as "muted".

**Comment by caarlos0** [comment]
@beorn7 @stuartnelson3 what do you guys think about renaming this form param to `muted`?

**Comment by beorn7** [comment]
We could do that. Or we create a new parameter `showInhibited` to give the UI the option to select those separately even if we wouldn't use it for now.

**Comment by brancz** [comment]
Yes I think it would be good to be able to view inhibited, silenced, and firing alerts in all combinations.

**Comment by stuartnelson3** [comment]
+1 to beorn's suggestion. Keep it flexibile in the API even if we don't make it an option from the UI.

**Comment by caarlos0** [comment]
@stuartnelson3 @beorn7 @brancz done!

**Comment by beorn7** [comment]
@stuartnelson3 I'd better leave the code level review to you. (One day, I'll get familiar with Alertmanager internals, but not today…)

File: api/api.go [hunk 1]
```diff
@@ -343,6 +359,10 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 			continue
 		}
 
+		if !showInhibited && len(status.InhibitedBy) != 0 {
```
Comment (brancz) [inline api/api.go]: still hoping that we'll one day have a real index around this :slightly_smiling_face:

**Comment by caarlos0** [comment]
> Generally looks good so far, but I would expect that I can toggle the front end to show me the inhibited alerts. As far as I can tell from this PR, this only handles the API filtering part. (if that is meant to be done in a subsequent PR, I'm fine with that as well)

Yeah, as far as I understood the original issue was about the backend only. I can change the frontend as well if required...

**Comment by beorn7** [comment]
@w0rm will take care of the frontend part (IIRC).

**Comment by brancz** [comment]
In that case, this lgtm

**Pull request merged**